fabfile.py

```diff
@@ -26,10 +26,12 @@
 
 
 @task
-def migrate(name_config=None, name_instance=None):
+def migrate(name_config=None, name_instance=None, debug=False):
     """
         :name_config - name of config yaml-file, example 'config.yaml'
     """
+    if debug:
+        utl.configure_logging(10)
```

Comment:
`10` is not readable magic number. Please replace with some readable variable name (e.g. `LOG_DEBUG=10`)